rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -458,8 +458,8 @@ static abstract class ComponentMixin extends Component {
                 "setAdditionalData",
                 "mainLicenseIdsIterator",
                 "setBinaryDownloadurl",
-                "setBinaryDownloadurl",
```

Comment:
Is this line removed correctly ?

Reply:
There were two occurrences of `setBinaryDownloadurl`, so I removed one, and replace it with `subscribersIterator`.